src/components/LuxInputText.vue

```diff
@@ -249,7 +249,7 @@ export default {
   },
   methods: {
     inputvaluechange(value) {
-      this.$emit("inputvaluechange", value)
+      this.$emit("update:value", value)
```

Comment:
It looks like the [datepicker is binding to the `inputvaluechange` event](https://github.com/pulibrary/lux-design-system/blob/9d155b87bb4294887112ef7f701849a9a398155a/src/components/LuxDatePicker.vue#L45).  Could we update that too, or emit both events?